src/components/NostoProduct/store.ts

```diff
@@ -121,3 +139,8 @@ export function injectStore(element: HTMLElement, cb: Callback) {
 }
 
 export type Store = ReturnType<typeof createStore>
+
+function getSkuValue<T extends keyof Sku>(skus: Sku[], key: T) {
+  const values = new Set(skus.map(sku => sku[key]))
+  return values.size === 1 ? [...values][0] : undefined
```

Comment:
This line is a bit confusing. We get the SKU objects of the select SKU Ids from the state. This method gets the array of SKU objects and extracts the info based on the supplied key. But I am not able to understand why we expect the line 144 to return only one element for the key (condition at line 145). Can't multiple selected SKU object have, for example, price?

Reply:
I have two questions now
1. Do we have the data for these use cases available from the server? If not, I am fine with adding it separately. 
2. Can you please add tests to verify where we could have multiple SKUs selected (NostoSkuOptions)?
3. Can you please add a Dev mode for this use case?